randovania/game_description/pickup/pickup_migration.py

```diff
@@ -199,6 +200,21 @@ def _migrate_v12(pickup_data: dict, game: RandovaniaGame) -> None:
         category.pop("is_broad_category", None)
 
 
+def _migrate_v13(pickup_data: dict, game: RandovaniaGame) -> None:
+    standard_pickups = pickup_data["standard_pickups"]
+    for pickup in standard_pickups.values():
+        if "must_be_starting" in pickup:
+            pickup["starting_condition"] = StartingEnum.MUST_BE_STARTING
+            pickup.pop("must_be_starting")
+        else:
+            pickup["starting_condition"] = StartingEnum.CAN_BE_STARTING
+    if game == RandovaniaGame.METROID_PRIME_ECHOES:
+        banned_starting_items = ["Cannon Ball", "Double Damage", "Unlimited Beam Ammo", "Unlimited Missiles"]
```

Comment:
You may need to do a change in the generator, so that it will never randomly use these as a starting item
(Place im commenting here is unrelated)